### ISSUE TITLE
php error

### ISSUE THREAD

**Issue by Bortus-AI** [erdesigns-eu/M3U2STRM#4]
How do I fix this
PHP Warning:  Undefined array key "tvg_name" in /mnt/mpathaj/navy/M3U2STRM/m3u2strm.php on line 191

**Comment by erdesigns-eu** [comment]
This error possibly occurs because you are parsing a playlist without tag-name in the playlist, so you can run the function convert_strm with as third parameter "name", this defaults to "tag_name" but if this isn't in the parsed playlist, it gives this error. I don't know if you download the m3u as m3u_plus (e.g. http://url.com/get.php?username=user&pass=pass&type=m3u_plus) because the default type=m3u doesn't include tag-name tags.

**Comment by Bortus-AI** [comment]
Yeah my m3u file doesn't have tags. 

This is what it has in it

https://urlxxxx/xxxxxxxxxxxx
#EXTINF:0 group-title="TV VOD",HD : TVSHOWNAME
#EXTGRP:TV VOD

**Comment by erdesigns-eu** [comment]
Okay yes i see, if you want i can rewrite it to support your playlist if you can send me a copy (replace the user and pass in the m3u file).

**Comment by Bortus-AI** [comment]
> Okay yes i see, if you want i can rewrite it to support your playlist if you can send me a copy (replace the user and pass in the m3u file).

Yes that would be amazing thank you. Do you have an email i can send it to?

**Comment by erdesigns-eu** [comment]
Yes you can send it to ernstreidinga85@gmail.com, ill look at it tomorrow and ill post the update

**Comment by Bortus-AI** [comment]
> Yes you can send it to, ill look at it tomorrow and ill post the update

Thank you! Email sent

**Comment by Bortus-AI** [comment]
> Yes you can send it to [ernstreidinga85@gmail.com](mailto:ernstreidinga85@gmail.com), ill look at it tomorrow and ill post the update

Wanted to see if you have had time to find a fix for this. Thanks

**Comment by Bortus-AI** [comment]
Any update?
